src/routes/Home/index.tsx

```diff
@@ -303,16 +303,25 @@ const Home = ({ videoId }: { videoId: string }) => {
   let datePublished = selectedVideo ? selectedVideo.snippet.publishedAt.toString() : 'NA';
   datePublished = datePublished.replace('T', ' Time: ').replace('Z', '');
 
+  //Simulating a volunteer
+  //let groups = ["Super User", "Volunteer"]
+  console.log(currentUser?.groups);
+
   return (
     <div>
       <Grid>
         <Grid.Row>
-          <Grid.Column style={{ marginLeft: 5 }} width={11}>
+          <Grid.Column style={{ marginLeft: 5 }} width={10}>
             {!loadingSelectedVideo ? (
               <div ref={setVideoColumnRef as any}>
                 {selectedVideo ? (
                   isLandingPage ? (
-                    <LandingPage user={currentUser ? (currentUser as any).firstName : 'Guest'} />
+                    <LandingPage
```

Comment:
For future reference, passing properties into components is fine, however if the property is already managed by Redux (such as the current user) it might be cleaner to retrieve the data from Redux within the component rather than passing it as a property. For example, see:

https://github.com/cusac/mbt.guide/blob/e81cef8e003e31e0b259652da9ca12457e0213d4/src/components/SegmentViewer.tsx#L11

If you like at some point we can do an overview of Redux (state management) within this app. The basics would not take long to cover.

Reply:
It is done. Thanks a lot for the hind. Yes, I need to learn more.